### PR TITLE
Make mix test run localy

### PR DESCRIPTION
**Pull request by ishikawa**
I fixed some flaws in `mix test`.

1. Set dummy environment variables `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to make AuthCache work.
2. Use `Bypass` to mock remote access.

### CODE REVIEW

File: test/test_helper.exs [hunk 1]
```diff
@@ -1 +1,5 @@
+# Configure access key and secret key with dummy value to make AuthCache work.
+System.put_env("AWS_ACCESS_KEY_ID", "dummy")
+System.put_env("AWS_SECRET_ACCESS_KEY", "dummy")
```
Comment (benwilson512) [inline test/test_helper.exs:3]: It would be better to set these in `test.exs` directly.

Comment (ishikawa) [reply test/test_helper.exs:3]: Do you mean `config/test.exs`?  If so, agreed.

Comment (benwilson512) [reply test/test_helper.exs:3]: I do yes.

Comment (ishikawa) [reply test/test_helper.exs]: Fixed in 6e7307b

**Pull request merged**